vsdeband/debanders.py

```diff
@@ -429,23 +279,20 @@ def f3k_deband(
         | kwargs
     )
 
-    debanded = core.neo_f3kdb.Deband(
+    logger.debug("vszip.Deband params: thr=%s, grain=%s %s", thr, grain, kwargs)
```

Comment:
accidental debug statement left in

Reply:
*logging* here is the keyword. This is not accidental and will not appear under normal circumstances.